tck/src/test/java/org/eclipse/microprofile/jwt/tck/util/TokenUtilsMiscMethodsTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.tck.util;
+
+import static org.eclipse.microprofile.jwt.tck.TCKConstants.TEST_GROUP_UTILS;
+
+import java.security.KeyPair;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Validation of the TokenUtils methods
+ */
+public class TokenUtilsMiscMethodsTest {
+
+    @Test(groups = TEST_GROUP_UTILS, description = "Used to generate a 2048 bit length key testing pair")
+    public void testKeyPairGeneration2048Length() throws Exception {
```

Comment:
Not very useful as this just test the JDK generation of an RSA key.

Reply:
Yeah, I was tempted to remove altogether but well, it is testing that `TokenUtils` is producing it correctly

Reply:
The other thing this test is testing is that `2048` is enforced, same for the other test